SpatialGDK/Source/SpatialGDK/Private/Utils/RPCContainer.cpp

```diff
@@ -71,7 +70,15 @@ namespace
 			*TimeDiff.ToString(),
 			*ERPCResultToString(ErrorInfo.ErrorCode));
 
-		if (TimeDiff.GetTotalSeconds() > FRPCContainer::SECONDS_BEFORE_WARNING)
+		const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
+		check(SpatialGDKSettings != nullptr);
+		float SecondsBeforeWarning = SpatialGDKSettings->RPCQueueWarningDefaultTimeout;
+		if (const float* CustomSecondsBeforeWarning = SpatialGDKSettings->RPCQueueWarningTimeouts.Find(ErrorInfo.ErrorCode))
```

Comment:
I would prefer if this logic lives inside SpatialGDKSettings, i.e.
```
float USpatialGDKSettings::GetSecondsBeforeWarning(ERPCResult Result);
```

Reply:
Code updated